[Docs]: `aws_eks_cluster.bootstrap_self_managed_addons` must be set to `false` for EKS Auto Mode (#40483)
hashicorp/terraform-provider-aws: website/docs/r/eks_cluster.html.markdown

### EKS Cluster with EKS Auto Mode

> ~> **NOTE:** When using EKS Auto Mode `compute_config.enabled`, `kubernetes_network_config.elastic_load_balancing.enabled`, and `storage_config.block_storage.enabled` must *ALL be set to `true`. Likewise for disabling EKS Auto Mode, all three arguments must be set to `false`. Additionally, `bootstrap_self_managed_addons` must be set to `false`, as this is a required parameter for enabling EKS Auto Mode.

Reviewer: ```suggestion
~> **NOTE:** When using EKS Auto Mode `compute_config.enabled`, `kubernetes_network_config.elastic_load_balancing.enabled`, and `storage_config.block_storage.enabled` must *ALL be set to `true`. Likewise for disabling EKS Auto Mode, all three arguments must be set to `false`. Enabling EKS Auto Mode also requires that `bootstrap_self_managed_addons` is set to `false`.
```
